src/libraries/System.Private.CoreLib/src/System/MemoryExtensions.Globalization.cs

```diff
@@ -35,6 +29,16 @@ public static bool Contains(this ReadOnlySpan<char> span, ReadOnlySpan<char> val
             return IndexOf(span, value, comparisonType) >= 0;
         }
 
+        /// <summary>
+        /// Returns a value indicating whether the specified span contains <see cref="char.IsWhiteSpace(char)">any
+        /// white-space characters</see>, and returns <see langword="true"/> if found. If not found, returns
+        /// <see langword="false"/>.
+        /// </summary>
+        /// <param name="span">The source span.</param>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Does this need `AggressiveInlining`? It should be consistent with `IsWhiteSpace` since their IL are equivalent in complexity.